lib_xcore_math/lib_build_info.cmake

```diff
@@ -0,0 +1,132 @@
+set(LIB_NAME lib_xcore_math)
+
+set(LIB_XMATH_PATH ${XMOS_SANDBOX_DIR}lib_xcore_math/lib_xcore_math)
+include(${XMOS_SANDBOX_DIR}lib_xcore_math/etc/build_options.cmake)
```

Comment:
Why no slash between `${XMOS_SANDBOX_DIR}` and `lib_xcore_math`?
I would expect this is go wrong if XMOS_SANDBOX_DIR is set without a trailing slash - I don't usually have a slash on the end when I set it.